tests/unit/utils/test_path.py

```diff
@@ -267,11 +266,80 @@ def test_existing_binary_in_windows_pathext(self):
             # Let's patch os.environ to provide a custom PATH variable
             with patch.dict(os.environ, {'PATH': os.sep + 'bin',
                                          'PATHEXT': '.COM;.EXE;.BAT;.CMD;.VBS;'
-                                         '.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY'}):
+                                                    '.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY'}):
                 # Let's also patch is_windows to return True
                 with patch('salt.utils.platform.is_windows', lambda: True):
                     with patch('os.path.isfile', lambda x: True):
                         self.assertEqual(
                             salt.utils.path.which('this-binary-exists-under-windows'),
                             os.path.join(os.sep + 'bin', 'this-binary-exists-under-windows.CMD')
                         )
+
+
```

Comment:
This needs to be marked as a `@DestructiveTest` because it modifies the file system